infrastructure/zk/src/down.ts

```diff
@@ -4,7 +4,7 @@ import * as utils from './utils';
 export async function down() {
     await utils.spawn('docker compose down -v');
     await utils.spawn('docker compose rm -s -f -v');
-    await utils.spawn('docker run --rm -v ./volumes:/volumes postgres:14 bash -c "rm -rf /volumes/*"');
+    await utils.spawn('docker run --rm -v volumes:/volumes postgres:14 bash -c "rm -rf /volumes/*"');
```

Comment:
`docker --version`? I am pretty sure this works on 24.0.7

Reply:
Updating Docker solved the issue, nevertheless I think that it would be better to have absolut paths in order to avoid this kind of issues. What do you think?

Reply:
it is fishy because it is not an absolute path.

Reply:
Hey! They are two different options to define a relative path, but the second one works for multiple docker versions, so I think it is better to avoid unexpected errors.

IMH I think we should keep the second option that @ilitteri suggested, or raise a proper error otherwise.

Reply:
It does not do what it is supposed to do, you can start a network with `zk init`, and then run `zk down` and then `ls ./volumes`, with this patch you will see this:

```
exa ./volumes/
geth  postgres  prysm
```
so it doesn't actually clean the directory although it is supposed to.